saleor/checkout/views/__init__.py

```diff
@@ -195,6 +195,25 @@ def update_cart_line(request, cart, variant_id):
     return JsonResponse(response, status=status)
 
 
+@get_or_empty_db_cart()
+def clear_cart(request, cart):
+    """Clear cart"""
+    from ..utils import update_cart_quantity
+
+    if not request.is_ajax():
+        return redirect('cart:index')
+
+    lines = cart.lines.all()
+    for line in lines:
+        line.delete()
+
+    update_cart_quantity(cart)
+
+    response = {'numItems': 0}
+
+    return JsonResponse(response, status=200)
```

Comment:
As far as I remember default status is `200`

Also, this function is stretched up a little, we could tighten it up :)